Sources/Sentry/SentryOptions.m

```diff
@@ -20,8 +20,8 @@ @implementation SentryOptions
 {
     return @[
         @"SentryCrashIntegration", @"SentryAutoBreadcrumbTrackingIntegration",
-        @"SentryAutoSessionTrackingIntegration", @"SentryOutOfMemoryTrackingIntegration",
-        @"SentryPerformanceTrackingIntegration"
+        @"SentryAutoSessionTrackingIntegration", @"SentryAppStartTrackingIntegration",
```

Comment:
L: Not really relevant to this PR necessarily but having 2 items per line makes 'diff' harder to scan.
I'd suggest having an item per line. if possible the last item should include a `,`

Reply:
I agree, but the code formatting changes it back to this.